### ISSUE TITLE
Support GNU toolchain on Windows 

### ISSUE THREAD

**Issue by m-hilgendorf** [astraw/ipp-sys#2]
The IPP libraries should be compatible with the GNU toolchain on Windows. I have a local fork running, all that should be needed is this line in `ipp-sys-build-help`: 

```rust 
fn ipp_build_inner(libname: String, libdir_base: PathBuf, target: String, link_type: &str) {
    assert!(link_type == "static" || link_type == "dylib");
    let mut libdir = libdir_base.clone();
    match target.as_ref() {
    /* ... */
        "x86_64-pc-windows-gnu" => {
            libdir = libdir.join("intel64");
        }
    /* ... */
    }
``` 

I haven't tested this extensively but it compiles, links, and runs successfully on my machine.

**Comment by astraw** [comment]
Thanks, can you check if b0e7f4a540cb7b7971dd249fa32ff615ebe6e761 works for you?

**Comment by m-hilgendorf** [comment]
Yes, thank you!

**Comment by astraw** [comment]
OK, thanks for checking. I just released ipp-sys-build-help 0.1.1 with this fix. I didn't change all the package requirements to avoid churn for this small change. You can do `cargo update` to get this latest version.
